README.md

```diff
@@ -4,15 +4,15 @@
 
 go-github is a Go client library for accessing the [GitHub API v3][].
 
-go-github requires Go version 1.8 or greater.
+go-github requires Go version 1.9 or greater.
 
 If you're interested in using the [GraphQL API v4][], the recommended library is
 [shurcooL/githubv4][].
 
 ## Usage ##
 
 ```go
-import "github.com/google/go-github/github"
+import "github.com/google/go-github/github/v18"
 ```
```

Comment:
Given that modules were released very recently and are still experimental/optional, perhaps it's a good idea to expand this to have two sections: one for traditional GOPATH workspaces (with `import "github.com/google/go-github/github"`), and another section for module-aware environments (with `import "github.com/google/go-github/v18/github"`).

Reply:
@dmitshur yes, and probably should include `go get` part too?